### ISSUE TITLE
Mileage should be below the heading for each result

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12910]
## Description
Screen reader users have the ability to navigate by headings on the page, they can easily jump directly to any one of these headings. Currently, the mileage (distance from the searched location) for each result is listed above the heading. This means that if a screen reader user jumps directly to a header for a results, then start reading from there, they will miss this mileage information.  By moving this information below the heading, this will no longer be missed when reading the location information for a result.

### Screenshot
Example screenshot using current locator and markup as the base:
![image](https://user-images.githubusercontent.com/106678594/224135110-fed5f85d-50ef-49ca-9c07-85d35bfbe303.png)

## Acceptance Criteria

- [ ] Mileage is displayed after the heading both visually and in the DOM

**Comment by laflannery** [comment]
@thejordanwood
